apps/wistia-videos/src/components/Field.tsx

```diff
@@ -75,6 +89,22 @@ const Field = (props: FieldProps) => {
     setNewValues(updatedIds);
   };
 
+  const getProjectIds = async (
+    excludedProjects: Project[],
+    bearerToken: string
+  ): Promise<string[]> => {
+    const projects = await fetchProjects(bearerToken);
+    console.info('Succesfully fetched the Wistia projects.');
+    // Map through projects and return ids to retrieve all the videos from each project. Filter out the projects selected to be excluded
+    return projects
+      .map((item: WistiaProject) => item.hashedId)
+      .filter((id: string) => {
+        const include =
```

Comment:
Maybe I'm thinking about this wrong, but why can't you just call `excludedProjects.includes(id)` which will return  a boolean if the id is in that array or not

Reply:
Good question, I didn't write this code initially (pasted it from what was there elsewhere in the file). Do you want me to change to use `includes`? I know it's not ideal where it is now, but it is working.

Reply:
I think it makes it simpler but if you are worried about regression, I think it is fine to leave as is!